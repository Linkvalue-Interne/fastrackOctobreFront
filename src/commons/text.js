import styled from '@emotion/styled';

export const H3 = styled.h3`
font-size: 1.7em;
padding: 1rem;
`;

export const P = styled.p`
font-size: 1.2em;
`;

export const SpanBold = styled.span`
font-weight: bold;
text-transform: uppercase;
`;
