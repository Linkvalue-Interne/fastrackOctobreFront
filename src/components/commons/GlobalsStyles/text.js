import styled from '@emotion/styled';
import { phoneQuery } from '../../../services/media-queries';

// You can use this list to import one or all element below
// { H3, P, SpanBold }

export const H3 = styled.h3`
font-size: 1.4em;
padding: 1rem;
font-size: ${(props) => props.fontSize};
`;

export const P = styled.p`
margin-top: 1em;
font-size: 1rem;
width: ${(props) => props.width};
font-weight: ${(props) => props.fontWeight};
margin: ${(props) => props.margin};
padding: ${(props) => props.padding};
${phoneQuery`
  font-size: 4vmin;
`};

`;

export const SpanBold = styled.span`
font-weight: bold;
text-transform: uppercase;
`;

export const WhiteP = styled.p`
  color: ${(props) => props.theme.colors.accentExtraLight};
  font-weight: 300;
  font-size: 1rem;
  margin-left: 0.5rem;
`;
