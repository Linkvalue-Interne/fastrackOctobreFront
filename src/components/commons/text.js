import styled from '@emotion/styled';

export const H3 = styled.h3`
font-size: 1.4em;
padding: 1rem;
`;

export const P = styled.p`font-size: 1rem`;

export const SpanBold = styled.span`
font-weight: bold;
text-transform: uppercase;
`;
