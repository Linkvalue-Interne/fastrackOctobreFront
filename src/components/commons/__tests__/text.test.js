import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-emotion';

import { H3, P, SpanBold } from '../text';

expect.addSnapshotSerializer(serializer);

describe('cardPartnerCSS', () => {
  it('renders with correct styles', () => {

    const tree = renderer.create(
      <SpanBold>
        <H3 />
        <P />
      </SpanBold>,
    )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
