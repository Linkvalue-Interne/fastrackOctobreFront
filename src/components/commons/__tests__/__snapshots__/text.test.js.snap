// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardPartnerCSS renders with correct styles 1`] = `
.emotion-2 {
  font-weight: bold;
  text-transform: uppercase;
}

.emotion-0 {
  font-size: 1.4em;
  padding: 1rem;
}

.emotion-1 {
  font-size: 1rem;
}

<span
  className="emotion-2"
>
  <h3
    className="emotion-0"
  />
  <p
    className="emotion-1"
  />
</span>
`;
